uavcan/node/435.ExecuteCommand.0.1.uavcan

```diff
@@ -0,0 +1,68 @@
+#
+# Instructs the server node to execute or commence execution of a simple predefined command.
+# All standard commands are optional; i.e., not guaranteed to be supported by all nodes.
+#
+
+# Standard pre-defined commands are at the top of the range (defined below).
+# Vendors can define arbitrary vendor-specific commands in the bottom part of the range (starting from zero).
+# Vendor-specific commands must not use identifiers above 32767.
+uint16 command
+
+# Reboot the node.
+# Note that some standard commands may or may not require a restart in order to take effect; e.g., factory reset.
+uint16 COMMAND_RESTART = 65535
+
+# Shut down the node; further access will not be possible until the power is turned back on.
+uint16 COMMAND_POWER_OFF = 65534
```

Comment:
should this be `COMMAND_ENTER_LOW_POWER` or `COMMAND_SLEEP` instead?

Reply:
I think "power off" conveys the intent better. This is mostly added to support on-board equipment that is permanently connected to the on-board power supply that should be able to somehow disable its own functionality for power conservation purposes when the vehicle is not in use. I agree that in most cases "low-power mode" or "sleep mode" would be adequate, but I don't want to impose any restrictions on how the power conservation feature should be actually implemented, hence I chose a very generic term "power off".